algosdk/abi/method.py

```diff
@@ -69,14 +72,14 @@ def get_selector(self):
         hash.update(self.get_signature().encode("utf-8"))
         return hash.digest()[:4]
 
-    def get_txn_calls(self):
+    def get_txn_calls(self) -> int:
         """
         Returns the number of transactions needed to invoke this ABI method.
         """
         return self.txn_calls
 
     @staticmethod
-    def _parse_string(s):
+    def _parse_string(s: str) -> list:
```

Comment:
```suggestion
    def _parse_string(s: str) -> List[str]:
```

Optional with a`from typing import List` above. I didn't suggest that in the `encod()` type hints, cause it was getting kind of verbose and I wasn't 100% what the `list` was made up of. But here it might make more sense

Reply:
Will do 👍